js/chat_app/src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { Button } from "@mui/material";
+import { NewUserForm } from "./components/NewUserForm/NewUserForm";
+import { NewChatForm } from "./components/NewChatForm/NewChatForm";
+import { Chats } from "./components/Chats/Chats";
+
+import "./App.css";
+
+function App() {
+  const [isCreateUserOpen, setIsCreateUserOpen] = useState(false);
+  const [isCreateChatOpen, setIsCreateChatOpen] = useState(false);
+
+  return (
+    <div className="App">
+      <Button variant="contained" onClick={() => setIsCreateUserOpen(true)}>
+        Create user
+      </Button>
+      <Button variant="outlined" onClick={() => setIsCreateChatOpen(true)}>
+        Create new chat
+      </Button>
+      <NewUserForm
+        open={isCreateUserOpen}
+        onClose={() => setIsCreateUserOpen(false)}
+      />
+      <NewChatForm
+        open={isCreateChatOpen}
+        onClose={() => setIsCreateChatOpen(false)}
+      />
+      <Chats />
```

Comment:
Maybe place this part in a separate component too?

Reply:
Which exactly component did you mean?
`NewChatForm` and `Chats `?

Reply:
I mean to add all these parts including Button, NewUserForm and NewChatForm in a separate component, so in App component we will have Header, LoginPage, separate component with Button, NewUserForm and NewChatForm (it's hard for me to name it in a proper way :)) and Chats. But I'm absolutely ok with the current variant!